test/1.4/expressions/T_endian_int.dml

```diff
@@ -240,4 +240,8 @@ method init() {
     assert (iarrp + var) == &iarr[2];
     assert (var + iarrp) == &iarr[2];
     assert (iarrp - var) == &iarr[0];
+
+    local uint32_be_t x = m();
```

Comment:
You could *consider* a comment here, e.g. jira ref or just "// implicit conversion from int works also for initializers". But I don't insist, the code is *almost* self-explaining.

Reply:
A jira ref, then. "implicit conversion from int works also for initializers" is not correct; it's not specific enough. The issue is exclusively when single-return function/method calls are used as initializers.

Reply:
Done.